packages/sanity/src/desk/types.ts

```diff
@@ -251,7 +251,7 @@ export interface PaneMenuItem extends DeskToolMenuItem {
   isDisabled?: boolean
   shortcut?: string
   selected?: boolean
-  tone?: 'primary' | 'positive' | 'caution' | 'critical'
+  tone?: ButtonTone
```

Comment:
Is there any benefit to this change? Feels a bit coincidental that it's the same set of tones as the Sanity UI button supports.

Reply:
You are right. There was a leftover `ButtonTone` imported there (see [`next`](https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/desk/types.ts#L3C17-L3C17)), so I thought a merge conflict had lost the `tone` change. Looks like its intentional to not use it there, even though its passed directly to Sanity UI components. I'll just remove this from the PR.